app/controllers/TaskController.scala

```diff
@@ -148,35 +149,46 @@ class TaskController @Inject() (val messagesApi: MessagesApi) extends Controller
       taskObjects: List[Fox[Task]] = requestedTasks.map(r => createTaskWithoutAnnotationBase(r._1))
       zipped = (requestedTasks, tracingReferences, taskObjects).zipped.toList
       annotationBases = zipped.map(tuple => AnnotationService.createAnnotationBase(
-                                                                taskFox = tuple._3,
-                                                                request.user._id,
-                                                                tracingReferenceBox=tuple._2,
-                                                                dataSetName))
+        taskFox = tuple._3,
+        request.user._id,
+        tracingReferenceBox = tuple._2,
+        dataSetName))
       zippedTasksAndAnnotations = taskObjects zip annotationBases
       taskJsons = zippedTasksAndAnnotations.map(tuple => Task.transformToJsonFoxed(tuple._1, tuple._2))
       result <- {
         val taskJsonFuture: Future[List[Box[JsObject]]] = Fox.sequence(taskJsons)
-        taskJsonFuture.map {taskJsonBoxes =>
+        taskJsonFuture.map { taskJsonBoxes =>
           bulk2StatusJson(taskJsonBoxes)
         }
       }
     } yield Ok(Json.toJson(result))
   }
 
-
+  private def validateScript(scriptId: Option[String])(implicit request: AuthenticatedRequest[_]): Fox[Option[String]] = {
+    scriptId match {
+      case Some(scriptIdString) =>
```

Comment:
Quick note on variable names: if in one function there is unpacking of an Option, we normally use the convention to call the option scriptIdOpt and unpack it with case Some(scriptId) =>…

Reply:
Ok will change that too.
Thanks for the fast response